recipes/qurro/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "qurro" %}
+{% set version = "0.6.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "9c911927525ed823d65cc017beb1ba8ab9dc7226986141bcb998233915c1620d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
```

Comment:
I don't think you need the `fn` line if it's specified in the `url` already.

Reply:
Thanks! I've removed the `fn` line in the latest recipe.